apps/meteor/ee/app/livechat-enterprise/server/business-hour/Helper.ts

```diff
@@ -21,9 +21,32 @@ const getAllAgentIdsWithoutDepartment = async (): Promise<string[]> => {
 	return agentIdsWithoutDepartment;
 };
 
+const getAllAgentIdsWithDepartmentNotConnectedToBusinessHour = async (): Promise<string[]> => {
+	const activeDepartmentsWithoutBusinessHour = (
+		await LivechatDepartment.findActiveDepartmentsWithoutBusinessHour({
+			projection: { _id: 1 },
+		}).toArray()
+	).map((dept) => dept._id);
+
+	const agentIdsWithDepartmentNotConnectedToBusinessHour = await LivechatDepartmentAgents.col.distinct('agentId', {
```

Comment:
Wonder if we should move this distinct to the model 🤔

Reply:
Let's do it